libs/community/langchain_community/chat_loaders/gmail.py

```diff
@@ -63,6 +64,11 @@ def _get_message_data(service: Any, message: Any) -> ChatSession:
     return ChatSession(messages=[starter_content, message_content])
 
 
+@deprecated(
+    since="0.1.15",
```

Comment:
```suggestion
    since="0.0.32",
```
I think this should be the `langchain_community` version